src/timesync/timesyncd-manager.c

```diff
@@ -1012,7 +1012,7 @@ static int manager_network_event_handler(sd_event_source *s, int fd, uint32_t re
 
         sd_network_monitor_flush(m->network_monitor);
 
-        changed = !!manager_network_read_link_servers(m);
+        changed = manager_network_read_link_servers(m);
```

Comment:
not sure how i feel about this one. the function returns negative on error... maybe do clarify that errors are treated like positive here, this should be changed to `!= 0` anyway...